api/src/modules/scenarios/cost-surface/cost-surface.facade.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@nestjs/common';
+import { Request, Response } from 'express';
+import { AdjustCostSurface } from '../../analysis/entry-points/adjust-cost-surface';
+import { ProxyService } from '../../proxy/proxy.service';
+
+@Injectable()
+export class CostSurfaceFacade {
+  constructor(
+    private readonly adjustCostSurfaceService: AdjustCostSurface,
+    private readonly proxyService: ProxyService,
+  ) {}
+
+  /**
+   * non blocking - will do job in "background"
+   *
+   * @param scenarioId
+   * @param request
+   */
+  convert(scenarioId: string, request: Request) {
```

Comment:
Is the intention to make this `async` but only placing it on the event loop without awaiting for it in the controller?

Reply:
yes, assuming we will do some processing there.

Reply:
i.e. there is no point in waiting within a controller as this is a "schedule job"-alike and we will process a file in an async way (but we cannot guarantee any reliable waiting time)